AssetInformationApi/V1/UseCase/SaveAssetUseCase.cs

```diff
@@ -0,0 +1,44 @@
+using AssetInformationApi.V1.Gateways;
+using AssetInformationApi.V1.UseCase.Interfaces;
+using System.Threading.Tasks;
+using Hackney.Shared.Asset.Infrastructure;
+using Hackney.Shared.Asset.Domain;
+
+namespace AssetInformationApi.V1.UseCase
+{
+    public class SaveAssetUseCase : ISaveAssetUseCase
+    {
+        private readonly IAssetGateway _gateway;
+
+        public SaveAssetUseCase(IAssetGateway gateway)
+        {
+            _gateway = gateway;
+        }
+
+        public async Task<Asset> PostAsync(Asset request)
+        {
+            var asset = new AssetDb()
```

Comment:
Instead of doing this, what if we do something like:
` return await _gateway.SaveAsset(asset.ToDatabase()).ConfigureAwait(false);
`
ToDatabase() is a method from the Factories

Reply:
used the Hackney.Shared.Asset.Factories ToDatabase()